### ISSUE TITLE
y_hook hooking other publics (FCNPC_OnCreate)

### ISSUE THREAD

**Issue by axi92** [Y-Less/YSI#66]
Im using the plugin FCNPC http://forum.sa-mp.com/showthread.php?t=428066 and try to hook FCNPC_OnCreate but it wont get called. Are there only the standard publics hookable or did I something wrong?
I hooked it at an included file but in that file I also hooked OnGameModeInit and that worked.

**Comment by Y-Less** [comment]
It only works on YSI 4.0, I totally rewrote the system to accomodate this.

**Comment by axi92** [comment]
4.0 is still in beta, am I right?

**Comment by Y-Less** [comment]
Yes, but frankly no release has ever been final, and parts of YSI 4.0 are the most stable and well tested that they've ever been!
